src/get_cubes/compile_metadata.Rmd

```diff
@@ -0,0 +1,44 @@
+---
+title: "compile_metadata"
+author: "Sander Devisscher"
+date: "2024-10-22"
+output: html_document
+---
+
+A script to compile the occurrence cubes from different ranks into a single cube.
+
+```{r libraries}
+library(readr)
+library(magrittr)
+library(dplyr)
+library(stringr)
+```
+
+```{r set ranks}
+ranks <- c("GENUS", "FAMILY", "SPECIES", "SUBSPECIES_FORM_VARIETY", "CLASS")
```

Comment:
I think it's impossible to automate even this step, right? Just to know what you think about it.